python/ray/util/client/common.py

```diff
@@ -31,8 +31,8 @@
 # 30 seconds because ELB timeout is 60 seconds
 GRPC_KEEPALIVE_TIME_MS = 1000 * 30
 
-# 20 seconds (gRPC) default
-GRPC_KEEPALIVE_TIMEOUT_MS = 1000 * 20
+# Long timeout because we do not want gRPC ending a connection.
+GRPC_KEEPALIVE_TIMEOUT_MS = 1000 * 600
```

Comment:
Does this mean that if ray.get does not return in 600 seconds this would fail?

Reply:
No, this is only for the keepalive itself. `ray.get` is 
1) An actual gRPC call (as opposed to a PING)
2) Has special logic to do short waits of ~5 seconds.